services/main/static-asset-server.mjs

```diff
@@ -31,10 +31,13 @@ class StaticAssetServer {
    */
   initialize() {
     // TODO: Restrict the visible pages.
-    this.app.use('/static', express.static(__dirname+'/../../'));
+    // Not sure if all static files are in sites
+    this.app.use('/static', express.static(__dirname+'/../../sites'));
+  }
+  run() {
     // Send the user to the game client page.
     this.app.get('/', function(req, res) {
-      res.redirect('/static/sites/game-client/client.html');
```

Comment:
redirect() works correctly by generating a 304 when I last tried.
If you want to send file directly, we'll lose the ability to import js/css through relative path. In that case, you might want to change client.html as well.